docs/terraformoptions.adoc

```diff
@@ -163,7 +163,12 @@ tags = {
 |false
 
 |`vcn_cidr`
-|The VCN's CIDR block. The CIDR block specified for the VCN must not overlap with the CIDR block of another network.
+|DEPRECATED: The VCN's CIDR block. Please use `vcn_cidrs` instead.
+|
+|10.0.0.0/16
```

Comment:
The default value should be in a list e.g.

`["10.0.0.0/16"]`

Reply:
Commit cd3ebcc8864da4d3d417b657a7c9d12662d9595f did just that.